ir_Lego_PF.cpp

```diff
@@ -44,3 +43,55 @@ void IRsend::sendLegoPowerFunctions(uint16_t data, bool repeat)
 }
 
 #endif // SEND_LEGO_PF
+
+//+=============================================================================
+#if DECODE_LEGO_PF
+bool  IRrecv::decodeLegoPowerFunctions (decode_results *results) {
+	static LegoPfBitStreamEncoder bitStreamEncoder;
+	uint16_t data   = 0;
+	int  offset = 1;
+
+	// Check SIZE
+	if (irparams.rawlen < 2 * (bitStreamEncoder.MESSAGE_BITS) - 1)  return false ;  // exclude STOP pause
+
+	// Check START_BIT Mark/Space
+	if (!MATCH_MARK (results->rawbuf[offset++], bitStreamEncoder.IR_MARK_DURATION - 58)) return false ;

```

Comment:
How did you derive the correction values 58, 100, 60, 62?
Would it make sense to use multiples of the 38kHz slices here? E.g. 53 and 105.